packages/webkit2gtk_4.rb

```diff
@@ -56,22 +57,55 @@ class Webkit2gtk_4 < Package
   depends_on 'mesa'
   depends_on 'openjpeg'
   depends_on 'pango'
+  depends_on 'valgrind' => :build
   depends_on 'wayland'
   depends_on 'woff2'
   depends_on 'wpebackend_fdo'
 
+  def self.patch
+    open('CMakeLists.txt', 'a') do |f|
+      f.puts 'set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")'
+      f.puts 'set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")'
+    end
+  end
+
   def self.build
     # This builds webkit2gtk4 (which uses gtk3)
     Dir.mkdir 'builddir4'
     Dir.chdir 'builddir4' do
-      # -flto breaks x86_64 builds
-      # system "env #{CREW_ENV_OPTIONS} \
       # Bubblewrap sandbox breaks on epiphany with
       # bwrap: Can't make symlink at /var/run: File exists
-      # ccache currently breaks gcc builds of webkit-gtk
-      system "cmake \
+      case ARCH
+      when 'x86_64'
+        system "CC=clang CXX=clang++ LD=lld NM=llvm-nm cmake \
+          -G Ninja \
+          #{CREW_CMAKE_OPTIONS.gsub('-flto', '').gsub('-ffat-lto-objects', '')} \
+          -DLTO_MODE=thin \
+          -DCMAKE_LINKER=$(which ld.lld) \
+          -DCMAKE_AR=$(which llvm-ar) \
+          -DCMAKE_RANLIB=$(which llvm-ranlib) \
+          -DCMAKE_SKIP_RPATH=ON \
+          -DENABLE_BUBBLEWRAP_SANDBOX=OFF \
+          -DENABLE_GAMEPAD=OFF \
+          -DENABLE_GLES2=ON \
+          -DENABLE_GTKDOC=OFF \
+          -DENABLE_INTROSPECTION=ON \
+          -DENABLE_MINIBROWSER=ON \
+          -DENABLE_VIDEO=ON \
+          -DENABLE_WAYLAND_TARGET=ON \
+          -DENABLE_WEB_AUDIO=OFF \
+          -DPORT=GTK \
+          -DUSE_GTK4=OFF \
```

Comment:
Why OFF?

Reply:
**GTK4=ON** switches the library to webkit2gtk_5.

Reply:
webkit2gtk_4 is explicitly built on GTK3, not GTK4.